.dockerignore

```diff
@@ -6,9 +6,6 @@
 media-repo*.yaml
 homeserver.yaml
 
-vendor/pkg
```

Comment:
ideally we'd actually switch this to just `vendor` to ignore in both files, for people who still have the old environment.